lib/puppet/version.rb

```diff
@@ -5,6 +5,11 @@
 # The version can be set programmatically because we want to allow the
 # Raketasks and such to set the version based on the output of `git describe`
 
+# Seems to be the minimum to get a vendored library without adding extra dependencies.
+vendor_dir = File.join([File.dirname(File.expand_path(__FILE__)), "vendor"])
+load "#{vendor_dir}/load_semantic_puppet.rb"
```

Comment:
I should have phrased my commit message better. That error log was requiring `puppet/vendor` and then calling `.load_vendored`. I don't like it either.